hippod/api_object_post.py

```diff
@@ -318,17 +346,20 @@ def try_adding_xobject(app, xobj):
     # object container file write should be delayed until everything
     # is sane.
     ok = is_obj_major_already_in_db(app, sha_major)
+    anchor_available = check_anchor(app, xobj)
+    # check whether anchor is in achievement to add correct initial lifetime of the object
     if not ok:
         # new entry, save to file
         # FULL update
-        hippod.store_container.save_new_object_container(app, sha_major, sha_minor, xobj,)
+        hippod.store_container.save_new_object_container(app, sha_major, sha_minor, xobj, anchor_available)
```

Comment:
a few lines further (l.362) is a function update_lifetime() which belongs to the posted object when no new (sub-)container is added and also needs return from check_anchor(). so i thougth call it once and pass it as argument would be the right way.
or is it still better to call it within save_new_object_container() etc. again?